vexe-llvm/src/test/kotlin/dev/supergrecko/vexe.llvm/utils/TestSuite.kt

```diff
@@ -0,0 +1,10 @@
+package dev.supergrecko.vexe.llvm.utils
+
+import dev.supergrecko.vexe.llvm.internal.contracts.Disposable
+import dev.supergrecko.vexe.test.TestSuite
+
+internal fun cleanup(item: Disposable) {
+    item.dispose()
+}
+
+internal open class TestSuite : TestSuite<Disposable>(::cleanup)
```

Comment:
Each sub-project is then able to declare it's own test suite which fits for that project.